infra/api/service/main.tf

```diff
@@ -135,6 +135,8 @@ module "service" {
       schema_name = local.database_config.schema_name
     }
   } : null
+
+  schedule_copy_oracle_data = true
```

Comment:
Wrapped in a boolean so it doesn't try to deploy step functions for the frontend service